src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -66,7 +67,25 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
   const legalNoteFieldName = `${baseFieldName}.${SpecificFileFieldNames.LegalNote}`;
   const rrsFieldName = `${baseFieldName}.${SpecificFileFieldNames.RightsRetentionStrategy}`;
 
-  const fileHasFunderRrs = file.rightsRetentionStrategy?.type === 'FunderRightsRetentionStrategy';
+  const isAcceptedFile = file.publisherAuthority === false;
+
+  const rrsStrategy = file.rightsRetentionStrategy.configuredType
+    ? file.rightsRetentionStrategy.configuredType
+    : customer?.rightsRetentionStrategy.type;
+
+  const isNullRrs = rrsStrategy === CustomerRrsType.NullRightsRetentionStrategy;
+  const isCustomerRrs = rrsStrategy === CustomerRrsType.RightsRetentionStrategy;
+  const isOverridableRrs = rrsStrategy === CustomerRrsType.OverridableRightsRetentionStrategy;
+
+  const fileHasFunderRrs = file.rightsRetentionStrategy.type === 'FunderRightsRetentionStrategy';
+  const fileHasCustomerRrs = file.rightsRetentionStrategy.type === 'CustomerRightsRetentionStrategy';
+  const fileHasOverriddenRrs = file.rightsRetentionStrategy.type === 'OverriddenRightsRetentionStrategy';
```

Comment:
burde nok sjekke for akseptert versjon også her

Reply:
Burde løses av denne: https://github.com/BIBSYSDEV/NVA-Frontend/pull/5262#discussion_r1520995533